src/lib/RoomConnection.ts

```diff
@@ -401,7 +423,7 @@ export default class RoomConnection extends TypedEventTarget {
                     isVideoEnabled: !!this.localStream?.getVideoTracks().find((t) => t.enabled),
                 },
                 deviceCapabilities: { canScreenshare: true },
-                displayName: "SDK",
+                displayName: this.displayName || "SDK",
```

Comment:
I think we should remove the "SDK" fallback, and just use nothing like the PWA

Reply:
I added this as the story looked weird without it (having an empty label) 🙈 . In PWA we have a `Guest` fallback. I can remove this though and fine-tune the story to have a fallback label content there to avoid "design" breaks.

Reply:
ah, I already added a `Guest` fallback for local participants, then I can just remove the fallback here 👍